src/main/java/by/yahor_kulesh/dao/TicketDAO.java

```diff
@@ -0,0 +1,119 @@
+package by.yahor_kulesh.dao;
+
+import by.yahor_kulesh.model.tickets.BusTicket;
+import by.yahor_kulesh.model.tickets.ConcertTicket;
+import by.yahor_kulesh.model.tickets.Ticket;
+import by.yahor_kulesh.utils.ObjectArray;
+
+import java.sql.*;
+import java.time.ZoneId;
+import java.util.UUID;
+
+public class TicketDAO{
+    private ResultSet rs = null;
+
+    public void insert(Ticket ticket){
+        try(
+                Connection con = DriverManager.getConnection("jdbc:postgresql://localhost:5432/my_ticket_service_db","postgres","postgres");
+                PreparedStatement stat = con.prepareStatement("INSERT INTO ticket(id,user_id,ticket_type,creation_date) VALUES (?,?,CAST(? AS ticket_type),?)")
```

Comment:
1. move connection configuration in separate `package config ConnectionConfig` class
```java
    // read values from file
    private String url = ...;
    private String user = ...;
    private String password = ...;
```
2.  into this class create `static` connection in method, for example:
```java
    public static Connection getConnection() throws SQLException {
        return DriverManager.getConnection(jdbcUrl, user, password );
    }
```
3. in your UserRepository or TicketRepository in all `try-with-resources` constructions call this method:
```java
try (Connection conn = ConnectionConfig.getConnection();
       PreparedStatement pstmt = conn.prepareStatement(sql)) {...}
```